korlibs-memory/build.gradle.kts

```diff
@@ -0,0 +1,21 @@
+import korlibs.*
+
+description = "Korlibs Memory"
+
+project.extensions.extraProperties.properties.apply {
+    applyProjectProperties(
+        "https://raw.githubusercontent.com/korlibs/korge/main/korlibs-memory",
+        "Public Domain",
+        "https://raw.githubusercontent.com/korlibs/korge/main/korlibs-memory/LICENSE"
+    )
+}
+
+dependencies {
+    commonMainApi(libs.kotlinx.coroutines.core)
+    commonMainApi(libs.kotlinx.atomicfu)
+    commonTestApi(libs.kotlinx.coroutines.test)
+    commonMainApi(project(":korlibs-math-core"))
+    commonMainApi(project(":korlibs-platform"))
+    commonMainApi(project(":korlibs-util"))
+    commonMainApi(project(":korlibs-crypto"))
+}
```

Comment:
We should evaluate those.

* kotlinx-coroutines are not used at all
* math-core might be used
* util and crypto probably not?

Reply:
util and math-core are used. We should avoid copiing stuff to private. I check the other two

Reply:
Okay, let's keep it as it is for now. We can optimize dependencies later.